Platform/CommonBoardPkg/Include/ConfigDataCommonDefs.h

```diff
@@ -12,7 +12,7 @@
 // and it is applicable for all platforms.
 
 // 0x050 to 0x05F is reserved for boot option tag
-#define  MAX_BOOT_OPTION_CFGDATA_ENTRY  16
+#define  MAX_BOOT_OPTION_CFGDATA_ENTRY  30

```

Comment:
Any reason to use 30 instread of 32?
Can you update comment as well? `// 0x050 to 0x05F is reserved for boot option tag`